### DISCUSSION TITLE
Save Icon for Toolbar

### DISCUSSION THREAD

**Discussion by froggiD** [coteditor/CotEditor#1496] [Ideas]
Unless I'm missing it, there is no SAVE icon/action available for the toolbar. You have to File/Save or Cmd/S. I'd love to see this added.

**Comment by 1024jp** [comment]
In macOS world, the save button is not put in the toolbar space. You may find no document-based applications provided by Apple has such icons in the toolbar.
CotEditor is designed to be macOS native, so it doesn't have the save icon  intentionally.
